packages/core/src/metrics/MetricsService.ts

```
@@ -93,10 +85,20 @@ export class MetricsService {
     });
   }
 
+  /**
+   * Collect metrics for all active metrics collectors
+   * Send data using metrics publisher
+   */
+  protected sendAppAndDeviceMetrics(): Promise<any> {
+    return this.publish(MetricsService.DEFAULT_METRICS_TYPE, []).catch((error) => {
```

Do we want to hide this error?